### ISSUE TITLE
Failed to resolve: com.wonderkiln:camerakit:0.13.5

### ISSUE THREAD

**Issue by YYKKKKXX** [wonderkiln/CameraKit-Android#646]
I already added `dependencies {
    implementation 'com.wonderkiln:camerakit:0.13.5'
}` to `build.gradle(:app)`

But it shows 
<img width="220" alt="1695808391880(1)" src="https://github.com/CameraKit/camerakit-android/assets/116107556/52f544f3-59a1-4ed5-babc-6c298e155892">

And later I watched a guiding video about camerakit on youtube,  then I tried`repositories {
    maven { url 'https://dl.bintrary.com/camerakit/camerakit-android' }
}` to `build.gradle(:app)`, but there was still something wrong,  and it did not go like the guiding video.

Have I do anything wrong? And what should I do now?

Thanks for reading.

**Comment by devahmedshendy** [comment]
@YYKKKKXX Were you able to solve this?

**Comment by harshakakarla77** [comment]
@YYKKKKXX I'm facing the exact same issue. Any leads?

**Comment by devahmedshendy** [comment]
@YYKKKKXX I got it working with beta3.10 of version 1, but not stable and sometimes work .. but still cannot make 0.13.4 or 0.13.5 working.

**Comment by rajesh221993** [comment]
**Add it in your root build.gradle at the end of repositories:**

	
		repositories {
			mavenCentral()
			maven { url 'https://jitpack.io' }
		
	}

**Step 2. Add the dependency**

	dependencies {
	        implementation 'com.github.CameraKit:camerakit-android:v0.13.5-SNAPSHOT'
	}

**Comment by hosseinkarami-dev** [comment]
Still this problem exists...

**Comment by weeklynote** [comment]
已收到，我会尽快回复你！
                     ——刘劲字

**Comment by YYKKKKXX** [comment]
> @YYKKKKXX Were you able to solve this?

No, later I chose to use another open project named 'CameraView'.

**Comment by YYKKKKXX** [comment]
> @YYKKKKXX I'm facing the exact same issue. Any leads?

Sorry, I didn't solve the problem, later I chose to use another open project named 'CameraView'.

**Comment by wztek-dev** [comment]
https://github.com/CameraKit/camerakit-android/releases => i've download source from this link and build to .arr and using it in my project

**Comment by weeklynote** [comment]
已收到，我会尽快回复你！
                     ——刘劲字
